src/timeMachine/components/FunctionModeSelector.tsx

```diff
@@ -62,9 +84,7 @@ const FunctionSelector: FunctionComponent<Props> = ({
       handleSetAutoFunction(false)
       return
     }
-    const newFunctions = selectedFunctions.filter(f =>
-      autoFunctions.includes(f)
-    )
+    const newFunctions = savedFunctions.filter(f => autoFunctions.includes(f))
```

Comment:
same here about renaming `f` to `savedFunction` or something like that please?

Reply:
Done.